domain/domain.go

```diff
@@ -827,25 +827,34 @@ func (do *Domain) LoadBindInfoLoop(ctx sessionctx.Context) error {
 		return err
 	}
 
-	do.loadBindInfoLoop()
+	do.globalBindHandleWorkerLoop()
 	do.handleInvalidBindTaskLoop()
 	return nil
 }
 
-func (do *Domain) loadBindInfoLoop() {
+func (do *Domain) globalBindHandleWorkerLoop() {
 	do.wg.Add(1)
 	go func() {
 		defer do.wg.Done()
-		defer recoverInDomain("loadBindInfoLoop", false)
+		defer recoverInDomain("globalBindHandleWorkerLoop", false)
+		loadTicker := time.NewTicker(bindinfo.Lease)
+		defer loadTicker.Stop()
+		captureBaselineTicker := time.NewTicker(bindinfo.Lease)
+		defer captureBaselineTicker.Stop()
```

Comment:
To many defer here. Can we merge them?

Reply:
The `recoverInDomain` cannot be merged, or it cannot recover.

Reply:
So the `do.wg.Done()` can't merge either?

Reply:
Yes, it should be called after recover.

Reply:
OK👌